tests/testthat/test-write.R

```diff
@@ -7,13 +7,9 @@ test_that("strings are only quoted if needed", {
   expect_equal(ssv, "a\n,\n")
 })
 
-test_that("a literal NA is quoted", {
-  expect_equal(format_csv(data.frame(x = "NA")), "x\n\"NA\"\n")
-})
-
```

Comment:
This test still applies to the first edition, though. Instead of removing it, I recommend inserting `skip_if_edition_second()`, which is used a lot elsewhere in this test suite.

Reply:
Yes you're right I had forgotten about that